vrc-get-gui/app/projects/manage/page.tsx

```diff
@@ -78,6 +78,8 @@ type RequestedOperation = {
 } | {
 	type: "remove";
 	pkgId: string;
+} | {
+	type: "appliedBulkChanges"
```

Comment:
the requested operation is bulk changes so the `RequestedOperation` type name should be something like `bulkChanges` I think.

Reply:
used seperate types, bulkInstalled and bulkRemoved